lib/workers/repository/process/sort.ts

```diff
@@ -16,8 +16,10 @@ export function sortBranches(branches: Partial<BranchConfig>[]): void {
     if (a.prPriority !== b.prPriority) {
       return b.prPriority - a.prPriority;
     }
+    const aUpdateType = a.updateTypes ? a.updateTypes[0] : null;
+    const bUpdateType = b.updateTypes ? b.updateTypes[0] : null;
     const sortDiff =
-      sortOrder.indexOf(a.updateType) - sortOrder.indexOf(b.updateType);
+      sortOrder.indexOf(aUpdateType) - sortOrder.indexOf(bUpdateType);
```

Comment:
So this now depends on order of assigned update types? We need to think about this sorting